features/query/unexpected_query_type_name/feature.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+from temporalio import workflow
+from temporalio.client import RPCError, WorkflowHandle
+
+from harness.python.feature import Runner, register_feature
+
+
+@workflow.defn
+class Workflow:
+    def __init__(self):
+        self.be_done = False
+
+    @workflow.run
+    async def run(self) -> None:
+        await workflow.wait_condition(lambda: self.be_done)
+        return None
+
+    @workflow.signal(name="finish")
+    def finish_sig(self):
+        self.be_done = True
+
+
+async def checker(_: Runner, handle: WorkflowHandle):
+    try:
+        await handle.query("nonexistent")
+    except RPCError:
+        pass
+    else:
+        raise Exception("Query with nonexistent handler must fail")
```

Comment:
```suggestion
    try:
        await handle.query("nonexistent")
        raise Exception("Query with nonexistent handler must fail")
    except RPCError:
        pass
```

If you want to save a line :-)